logic/scripts/update_token_insight.py

```diff
@@ -321,9 +333,46 @@ def fetch_ogn_transactions():
             db.session.add(tx)
             db.session.commit()
 
+# Fetches wallet balance from API and stores that to DB
+def fetch_wallet_balance(wallet):
+    print "Checking the balance of wallet %s" % (
+        wallet,
+    )
+
+    url = "http://api.ethplorer.io/getAddressInfo/%s" % (wallet)
+    results = call_ethplorer(url)
+
+    contact = db_common.get_or_create(
+        db.session, db_models.EthContact, address=wallet
```

Comment:
Would it be worth setting the name so that we know what the wallet is for when we look at the data?
Like "Origin Protocol foundation reserver", "Origin Protocol team distribution", etc...

Reply:
This was already existing code, I just extracted some parts of it into a separate function. Looking at the model, we already have a name column, can do a model update here.